scripts/decrypt.sh

```diff
@@ -1,13 +1,20 @@
 #!/usr/bin/env bash
-# This script decrypts our test database 
-# WARNING: Edit decrypt.template.mustache not decrypt.sh which is a generated file
+# This script decrypts our test database
 # Bash3 Boilerplate. Copyright (c) 2014, kvz.io
+# You will need the environment variables CIRCLE_CI_KEY and CIRCLE_CI_IV
 
 set -o errexit
 set -o pipefail
 set -o nounset
 set -o xtrace
 
-openssl aes-256-cbc -K $encrypted_f3e772af0e32_key -iv $encrypted_f3e772af0e32_iv -in secrets.tar.enc -out secrets.tar -d
```

Comment:
The old approach had different keys when a new confidential bundle was uploaded. 
What is the strategy when someone wants to work on a new bundle in a branch but other builds need to use the old bundle?

Reply:
> The old approach had different keys when a new confidential bundle was uploaded.
> What is the strategy when someone wants to work on a new bundle in a branch but other builds need to use the old bundle?

I was wondering why the zip file had a folder in it, and I had suggested to Gary in another PR comment to remove it. But actually, I still don't get it. When files are extracted from the zip, they are extracted relative from the folder into your local file system. So if I were switching between branches, I would have two zip files, one for each branch, and extract from the appropriate one.

I feel like I'm missing something.

Reply:
Ya, I don't get the issue. One bundle for each branch. Just run bash encrypt.sh again

Reply:
I think the piece I was missing is that all branches are using the same key, so all bundles can be decrypted using the same key + iv value.